docs/source/api/pystiche_papers.li_wand_2016.rst

```diff
@@ -27,31 +27,149 @@
 .. |archive| replace:: Archive
 .. _archive: https://github.com/pmeier/CNNMRF/tree/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f
 
-Unfortunately, the parameters in the reference implementation differ from the parameters
-described in the paper. If ``impl_params is True``, the parameters from the reference
-implementation are used instead of the parameters from the paper. The following
-parts are affected:
+.. _table-li_wand_2016-impl_params:
 
-  - :func:`~pystiche_papers.li_wand_2016.image_pyramid`,
-  - :func:`~pystiche_papers.li_wand_2016.content_loss`,
-  - :func:`~pystiche_papers.li_wand_2016.style_loss`,
-  - :func:`~pystiche_papers.li_wand_2016.regularization`,
-  - :func:`~pystiche_papers.li_wand_2016.target_transforms`.
+Implementation parameters
+-------------------------
+
+Unfortunately, the the behavior in the reference implementation differ from what is
+described in the paper. The following parts are affected:
+
+- :class:`~pystiche_papers.li_wand_2016.FeatureReconstructionOperator`
+- :class:`~pystiche_papers.li_wand_2016.MRFOperator`
+- :class:`~pystiche_papers.li_wand_2016.TotalVariationOperator`
+- :class:`~pystiche_papers.li_wand_2016.target_transforms`
+
```

Comment:
The pyramid is also influenced by this parameter.

> num_steps = 100 if impl_params else 200
num_levels = 3 if impl_params else None